cmd/clock_generic.go

```diff
@@ -0,0 +1,14 @@
+// +build !test
+
+package cmd
+
+import "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+
+// Clock functions similarly to clock.Default(), but the returned value can be
+// changed using the FAKECLOCK environment variable if the 'test' build flag is
```

Comment:
I think we actually don't want to reuse the 'test' flag here, since it's used for unittests. We want this for the integrations tests. Maybe call it 'integration_test'?